mountaineer/client_builder/openapi.py

```diff
@@ -99,10 +130,20 @@ class Reference(BaseModel):
 
         model_config = {"populate_by_name": True}
 
+        @classmethod
+        def from_meta(cls, ref: str) -> "ContentDefinition.Reference":
+            # Workaround to pyright overriding the instance method with
```

Comment:
It's a bit verbose, but we add this from_meta convention for all internal Pydantic models that have an alias defined. Pyright https://github.com/microsoft/pyright/discussions/1782#discussioncomment-1920822 the method signature to just support the alias values, but our aliases here use reserved words within the Python runtime. We therefore have to first cast these attributes to a dictionary before allowing the initial pydantic validator to hydrate the instance.